smk-edit/static/components/edit-item-details-layer-esri-dynamic.js

```diff
@@ -4,6 +4,49 @@ export default importComponents( [
 ] ).then( function () {
     return vueComponent( import.meta.url, {
         props: [ 'itemId' ],
+        computed: {
+            hasLabels: function() {
+                const layer = this.$store.getters.configLayer( this.itemId );
+                if (layer.dynamicLayers) {
```

Comment:
Oops, missed this. If layer is undefined this will throw an exception. Same for the get and set functions below.